src/main/java/org/springframework/data/couchbase/repository/auditing/CouchbaseAuditingRegistrar.java

```diff
@@ -84,7 +86,7 @@ protected void registerAuditListenerBeanDefinition(BeanDefinition auditingHandle
 				.addConstructorArgValue(ParsingUtils.getObjectFactoryBeanDefinition(getAuditingHandlerBeanName(), registry));
 
 		registerInfrastructureBeanWithId(listenerBeanDefinitionBuilder.getBeanDefinition(),
-				AuditingEventListener.class.getName(), registry);
+				AuditingEntityCallback.class.getName(), registry);
```

Comment:
I think this just changes the name used in registering?  On line 84, the rootBeanDefinitions is still AuditingEventListener.class - and after this change, the AuditEventListener is still the listener class is being used.   And it already had an onApplicationEvent() that calls markedAudit().   The changed line above can even be changed to as below and it still works.  So I don't think this particular change is needed - and I don't think the AuditingEventCallback class is needed.

/*AuditingEntityCallback.class.getName()*/ "My_Friend_Dave", registry

Reply:
ok - I just looked at the ReactiveCouchbaseAuditingRegistrar - and it appears that the intention is to use the new AuditEntityCallback.class - both when creating the listenerBeanDefinitionBuilder and when registering.   And to no longer use the AuditEventListener.